tests/e2e/message_counts_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	//"log"
 	//"os"
-	"strconv"
+	//"strconv"
```

Comment:
Why not just remove the commented out stuff?

Reply:
removed